greetings: regularly print a string to the console every second. (#61141)
zephyrproject-rtos/zephyr: samples/basic/greetings/README.rst

> .. _greetings:
> 
> Greetings
> #########

Reviewer: ```suggestion
.. zephyr:code-sample:: greetings
   :name: Greetings
   :relevant-api: thread_apis

   Print greetings to the console once per second.
```

Overview
********

A simple sample that can be used with any :ref:`supported board <boards>` and
prints "Zephyr says 'Hi!'" to the console, once per second.

> It's an alternative to :ref:`blinky <blinky-sample>` for boards with no

Reviewer: ```suggestion
It's an alternative to :zephyr:code-sample:`blinky` for boards with no
```

controllable LEDs.

Building and Running
********************

This application can be built and executed on QEMU as follows:

.. zephyr-app-commands::

>    :zephyr-app: samples/greetings

Reviewer: ```suggestion
   :zephyr-app: samples/basic/greetings
```
